src/features/join/hooks/useUserHook.ts

```
@@ -32,26 +33,36 @@ export const useAgreeTerms = () => {
 export const useVerifyCertificationCode = () => {
   return useMutation({
-    mutationFn: (params: { phoneNum: string; certificationCode: string }) =>
-      verifyCertificationCode(params.phoneNum, params.certificationCode),
+    mutationFn: (params: { phoneNumber: string; certificationCode: string }) =>
+      verifyCertificationCode(params),
     onSuccess: () => {
       alert('인증에 성공했습니다.');
     },
-    onError: () => {
-      alert('인증번호가 일치하지 않습니다.');
+    onError: (error: AxiosError<any>) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)